i18n/en/pop_desktop_widget.ftl

```diff
@@ -69,11 +69,11 @@ page-dock = Dock
 page-main = General
 page-workspaces = Workspaces
 
-position-bottom = Bottom of the screen
-position-left = Along the left side
-position-right = Along the right side
+position-bottom = Bottom
+position-left = Left
+position-right = Right
 
-alignment-center = Center
+alignment-center = Middle
```

Comment:
I think "center" is more commonly used together with "left" and "right". To me it makes the options more coherent when you use "middle".